test/chisel2.py

```diff
@@ -0,0 +1,229 @@
+"""
+A simple client that uses the Python ACME library to run a test issuance against
+a local Boulder server. Usage:
```

Comment:
Can you adjust this comment so that it clarifies what the difference between `chisel.py` and `chisel2.py` is? They're both for running a test issuance against a local Boulder server but this one is for a local ACMEv2 enabled Boulder server :-)